js/genkit/tests/session_test.ts

```diff
@@ -361,22 +361,20 @@ describe('session', () => {
   });
 
   it('can start chat from a prompt', async () => {
-    const agent = ai.definePrompt(
-      {
-        name: 'agent',
-        config: { temperature: 1 },
-        description: 'Agent description',
-      },
-      '{{role "system"}} hello from template'
-    );
+    const agent = ai.definePrompt({
+      name: 'agent',
+      config: { temperature: 1 },
+      description: 'Agent description',
+      system: 'hello from template',
```

Comment:
What would happen if someone did:

```
system: 'hello {{role "user"}}this is user now'
```

Reply:
it will throw an error... `system` and `prompt` allow only one message. This would have to go under `messages`.